<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrivateChat</class>
 <widget class="QWidget" name="PrivateChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(48, 56, 65);</string>
  </property>
  <widget class="QListWidget" name="chatListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(238, 238, 238);
font: 75 bold italic 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(238, 238, 238);
font: 75 bold italic 9pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputTextBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(238, 238, 238);
font: 75 bold italic 9pt &quot;Comic Sans MS&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadFileButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(238, 238, 238);
font: 75 bold italic 9pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Upload File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadFileButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(238, 238, 238);
font: 75 bold italic 9pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Download File</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="downloadProgressBar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QComboBox" name="emojiComboBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(238, 238, 238);
font: 75 bold italic 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
